extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -110,14 +97,29 @@ define('ext.wikia.adEngine.slotTweaker', [
 		}
 	}
 
+	function getRecoveredIframe(slotName) {
+		var fallbackId = win._sp_.getElementId(document.querySelectorAll('div[id="' + slotName + '"] div')[1].id);
+		return doc.getElementById(fallbackId).querySelector('div:not(.hidden) > div[id*="_container_"] iframe');
+	}
+
 	function onReady(slotName, callback) {
 		var iframe = doc.getElementById(slotName).querySelector('div:not(.hidden) > div[id*="_container_"] iframe');
 
-		if (!iframe) {
+		if (!iframe && !win._sp_) {
```

Comment:
Please don't rely on `_sp_`. It's also defined when AdBlock is off (and recovery is enabled on wiki). Please check `ext.wikia.aRecoveryEngine.recovery.helper::isBlocking`